lib/rules/web/admin_console/4.2/operators.xyaml

```diff
@@ -0,0 +1,54 @@
+operatorhub_click_install:
+  elements:
+  - selector:
+      xpath: //a[@class="pf-c-button pf-m-primary co-catalog-page__overlay-create" and contains(., "Install")]
+    op: click
+
+create_operator_subscription:
+  action:
+    if_element:
+      selector:
+        xpath: //button[@id="dropdown-selectbox" and @class="pf-c-dropdown__toggle"]
+    ref: select_target_namespace
```

Comment:
I see `<project_name>` is passed in `goto_operatorhub_subscribe_page` so I think we don't need `select_target_namespace` any more

Reply:
Yes, I'll update the rules. Thanks.